### DISCUSSION TITLE
ODOH server odoh-koki-ams issue

### DISCUSSION THREAD

**Discussion by zucht** [DNSCrypt/dnscrypt-proxy#1887] [Potential issues]
When using odoh-koki-ams as my ODOH server i get the folowing error the last couple of days:

[ERROR] [odoh-koki-ams] didn't return an ODoH configuration - [404 Not Found]

Potential issue with this server? odoh-cloudfare works fine.

**Comment by kokial** [comment]
odoh-koki-ams  is working now
